spec/paypal_express/remote/baid_helpers.rb

```diff
@@ -0,0 +1,50 @@
+module BaidHelpers
```

Comment:
You should namespace it in the `Killbill` module. The chance of collisions is low here, but it's good practice (same below).